README.md

```diff
@@ -7,4 +7,4 @@ toggled via toolbar icon.
 
 You can install it via [the Chrome Web Store](https://chrome.google.com/webstore/detail/wikimediadebug/binmakecefompkjggiklgjenddjoifbb).
 
-There is also a [Firefox version](https://addons.mozilla.org/firefox/addon/wikimedia-debug-header/).
+There is also a [Firefox version](https://github.com/wikimedia/FirefoxWikimediaDebug).
```

Comment:
Just for consistency with the other direction, and because anyone who
purposefully went to github might be trying to see the code. Feel free to
choose either way, on both repositories. :)